### DISCUSSION TITLE
[Issue forms] Allow dropdowns to be displayed inline

### DISCUSSION THREAD

**Discussion by ShockedPlot7560** [community/community#37651] [Projects and Issues]
Currently, if we put a dropdown with a label in a issue form, the label will be displayed as the title, and the value of the dropdown as the text after this title.  
There should be a way to specify that you want an inline display as follows: 

**Actually:**
```markdown
## A label

The value
```
**What it should be possible to do:**
```markdown
A label: **the value**
```

**Comment by ossanna16** [comment]
Hi there @ShockedPlot7560 and welcome to our community! Thank you for the feedback 🙂

**Comment by Timmmy-nlb** [comment]
This would be extremely useful to be able to process data from the form in workflows! 
Or does anyone know another way that would lead to the same goal?
